modules/mobile/docs/openapi.yaml

```diff
@@ -195,16 +195,16 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Errors"
-  /v0/letters/{id}:
+  /v0/letters/{type}/download:
     post:
       summary: /v0/letters/{id}
       description: Returns requested letter for download
-      operationId: downloadLetterWithOptions
+      operationId: download
       security:
         - Bearer: [ ]
       parameters:
         - in: path
-          name: id
+          name: type
```

Comment:
Is the address passed in here?

Reply:
The address seems to be pulled in by the backend service. Even on .gov the current letters endpoints have no awareness of the address.

Reply:
Ah ok, I am remembering the discussion about the UX where the user will choose an address to be printed on the letter, but right, that is saved through the profile.